src/contribot/lib/slack_webhook.py

```
@@ -0,0 +1,55 @@
+        if "SecretString" not in response:
+            raise RuntimeError(f"Secret {arn} has no SecretString")
+        self._webhook_url = response["SecretString"].strip()
+
+    def post(self, text: str) -> None:
+        """Send ``text`` as a Slack message; raise RuntimeError on failure."""
+        request = Request(
+            self._webhook_url,
+            data=json.dumps({"text": text}).encode("utf-8"),
+            headers={"Content-Type": "application/json"},
+            method="POST",
+        )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)